README.md

```diff
@@ -0,0 +1,4 @@
+# Bookstore
+
+## Live Demo
+[Heroku](https://bookstore-roberto.herokuapp.com/)
```

Comment:
**Let's make your repo looks professional by applying all the 3 basic rules, kindly update your `README` file**.

Checkout out the below link for help, feel free to follow this [template](https://github.com/microverseinc/readme-template) from pathwright

![image](https://user-images.githubusercontent.com/31897434/92685488-b4c45680-f32f-11ea-9bea-222a21c096a6.png)

https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/